src/ucp/core/ucp_request.h

```diff
@@ -237,13 +245,14 @@ struct ucp_request {
                 } discard_uct_ep;
 
                 struct {
+                    ucs_ptr_map_key_t     sreq_id;     /* Send request ID */
                     uint64_t              remote_addr; /* Remote address */
                     ucp_rkey_h            rkey;        /* Remote memory key */
                     uint64_t              value;       /* Atomic argument */
-                    uct_atomic_op_t       uct_op;      /* Requested UCT AMO */
                 } amo;
 
                 struct {
+                    ucs_ptr_map_key_t sreq_id;   /* Send request ID */
```

Comment:
why do we need this in offload if it doesn't have error handling support?

Reply:
removed